src/test/java/org/openrewrite/java/logging/ParameterizedLoggingTest.java

```diff
@@ -65,6 +65,44 @@ static void method(Logger logger, String name) {
         );
     }
 
+    @Test
+    void exceptionAppended() {
+        rewriteRun(
+          spec -> spec.recipe(new ParameterizedLogging("org.slf4j.Logger error(..)", false)),
```

Comment:
Hmm; even if I change this line to run the SLF4J Yaml recipe, then the test is still green
```suggestion
          spec -> spec.recipeFromResources("org.openrewrite.java.logging.slf4j.ParameterizedLogging"),
```